EventBriteAssignment/Dockerfile

```
@@ -4,16 +4,16 @@ EXPOSE 80
 
 FROM mcr.microsoft.com/dotnet/core/sdk:2.1-stretch AS build
 WORKDIR /src
-COPY ["EventBriteCatalog/EventBriteCatalog.csproj", "EventBriteCatalog/"]
-RUN dotnet restore "EventBriteCatalog/EventBriteCatalog.csproj"
+COPY ["EventBriteAssignment/EventBriteAssignment.csproj", "EventBriteAssignment/"]
```

We changed the name of the project to EventCatalog from 3A. The folder is still called EventBriteAssignment though
```suggestion
COPY ["EventBriteAssignment/EventCatalog.csproj", "EventBriteAssignment/"]
```